docs/product-guides/secure-work-gs.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Secure Workforce Next Generation Getting Started
+id: secure-workforce-gs
+description: 'Secure Workforce Next Generation Getting Started'
+slug: /secure-workforce-gs
```

Comment:
I'm wondering if we should just leverage the existing "/secure-workforce" page rather than creating a new one. What do you think?

Reply:
Yeah, that makes sense. I forgot to check my changes in on thursday.